tee-worker/app-libs/stf/src/getter.rs

```diff
@@ -208,11 +208,11 @@ impl ExecuteGetter for PublicGetter {
 	fn execute(self) -> Option<Vec<u8>> {
 		match self {
 			PublicGetter::some_value => Some(42u32.encode()),
-			PublicGetter::nonce(who) => {
+			PublicGetter::nonce(who, tx_num) => {
 				let nonce = System::account_nonce(&who);
 				debug!("PublicGetter nonce");
 				debug!("Account nonce is {}", nonce);
```

Comment:
I think this message should print value which is being returned from function (`nonce + tx_num`)